### PR TITLE
replace n+1 calls in Boards and Backlog

### PR DESCRIPTION
**Pull request by pbruski**

### CODE REVIEW

File: src/main/kotlin/com/atlassian/performance/tools/jirasoftwareactions/webdriver/JavaScriptUtils.kt [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+import org.openqa.selenium.JavascriptExecutor
+import org.openqa.selenium.WebDriver
+
+class JavaScriptUtils {
+    companion object {
+        @JvmStatic
+        fun <T> executeScript(driver: WebDriver, js: String): T {
+            driver as JavascriptExecutor
+            return driver.executeScript(js) as T
```
Comment (dagguh) [inline src/main/kotlin/com/atlassian/performance/tools/jirasoftwareactions/webdriver/JavaScriptUtils.kt]: Let's suppress the unchecked cast warning

File: src/main/kotlin/com/atlassian/performance/tools/jirasoftwareactions/api/page/ViewBacklogPage.kt [hunk 2]
```diff
@@ -15,8 +16,8 @@ class ViewBacklogPage(
     )
 
     fun getIssueKeys(): List<String> {
-        return driver
-            .findElements(By.className("js-issue"))
-            .map { it.getAttribute("data-issue-key") }
+        return JavaScriptUtils.executeScript(driver,
```
Comment (dagguh) [inline src/main/kotlin/com/atlassian/performance/tools/jirasoftwareactions/api/page/ViewBacklogPage.kt]: Nice one. Alternatively we could have used:
```kotlin
driver
    .findElements(...)
    .take(10)
    .map { ... }
```
It'd convert the 1+n into 1+10.
But it's fine to convert it to a 1 (although heavier), if it's less overhead this way.
Is there an easy way of checking whether the 1+10 would cost more or less?

File: src/main/kotlin/com/atlassian/performance/tools/jirasoftwareactions/api/page/ViewBoardPage.kt [hunk 3]
```diff
@@ -16,8 +16,8 @@ class ViewBoardPage(
     )
 
     fun getIssueKeys(): List<String> {
-        return driver
-            .findElements(className("ghx-issue"))
-            .map { it.getAttribute("data-issue-key") }
+        return JavaScriptUtils.executeScript(driver,
```
Comment (dagguh) [inline src/main/kotlin/com/atlassian/performance/tools/jirasoftwareactions/api/page/ViewBoardPage.kt]: Nice, overhead reduction is always welcome. Let's changelog it!

**Comment by pbruski** [comment]
@dagguh I've measured it in chrome, going through 21000 elements and extracting their attributes took about 18ms. I am willing to bet that (1+10)x2 REST calls is going to take more :-)

**Comment by dagguh** [comment]
Awesome! Thanks for the numbers 🏆

**Pull request merged**